Strategies/ITVision.md

```diff
@@ -0,0 +1,168 @@
+3. [Digital-first and digitally enabled by design](#digital-first-and-digitally-enabled-by-design)
+4. [Modern tech, technology practices and information stewardship](#modern-tech-technology-practices-and-information-stewardship)
+5. [A digitally enabled public service](#a-digitally-enabled-public-service)
+6. [Good digital governance](#good-digital-governance)
+
+## 1. User-centred, service orientation
+
+Human Interfaces that citizens interact with (e.g. websites, web applications, and mobile applications) are rarely provided by ESDC.
+Instead, **ESDC does not compete with the private sector** and seeks to leverage it to drastically improve citizen's user experience as well as [promoting Canadian innovation and skills](https://www.theglobeandmail.com/business/commentary/article-the-essentials-for-canadas-digital-economy/).
+ESDC makes use of the public and private industries such as job promoting websites, banks, academic institutions, and employers, to offer its services via APIs (see [An open, collaborative, and accessible government](#an-open-collaborative-and-accessible-government)).
+ESDC establishes terms of service and conditions with private industries to assure its services are always equitable, accessible, and do not have barriers for users with disabilities.
+ESDC contributes to the [OneGC vision](https://gccollab.ca/groups/profile/154708/enonegc-ungcfrungc) by leveraging [Government of Canada's single online window](https://open.canada.ca/en/content/service-strategy-overview) instead of creating its own, removing the burden on the citizen to understand how the Government of Canada is internally administered via its multiple departments.
+
+Artificial Intelligence is being used to drastically **[reduce toil](https://landing.google.com/sre/sre-book/chapters/eliminating-toil/)**, greatly improving ESDC's commitment to citizens through its continuously improving service standards, enabling auto-enrolment by design, providing insights for service innovation, while ensuring ethical use of [automated decision-making](https://tbs-sct.gc.ca/pol/doc-eng.aspx?id=32592). Artificial Intelligence is a shared expertise between the Chief Information Officer (CIO), responsible for the technology infrastructure hosting AI software, and the Chief Data Officer (CDO), responsible for data sciences.
+
+All of ESDC's services are connected and interoperable to enable the **"tell us once"** goal.
+All of ESDC's services use a single account, tied to a single identity, in order to avoid multiple separate logins and use **[modern identity verification](https://gcn.com/articles/2019/03/25/mobile-verification.aspx)** to drastically improve identity theft protection for citizens.
+
+## 2. An open, collaborative, and accessible government
+
+The principles of open by default and inclusive by design are integrated into all new processes, systems, solutions, and services.
```

Comment:
Another benefit is hiring ... you can use top contributors to find good employees. They would already be familiar with the code base, and have already showed initiative, and we could see the quality of their work